ai-platform-unified/notebooks/unofficial/reduction_server/distributed-training-reduction-server.ipynb

```diff
@@ -0,0 +1,1272 @@
+{
```

Comment:
<p>Might want to include "Open in Colab" and Github buttons.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/417/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>